Dockerfile

```diff
@@ -0,0 +1,25 @@
+# kovri - I2P but in CPP
+# Copyright (c) 2015-2017, The Kovri I2P Router Project (see LICENSE.md)
+# docker build -t geti2p/kovri .
+# KOVRI_PORT=42085 && sudo docker run -p $KOVRI_PORT --env KOVRI_PORT=$KOVRI_PORT geti2p/kovri
+
+FROM alpine:3.5
+RUN apk add --update --no-cache -t .kovri-dev g++ make cmake binutils boost-dev libressl-dev
+ENV KOVRI_HOST 0.0.0.0
+ENV KOVRI_PORT 24085
+ADD . kovri/
```

Comment:
I don't have strong feeling either way. I am going to do some research and see if there is a docker-community consensus. :)

Right now for dockerhub I have it pointed towards a repo under my account, but you have a good point.

I'll do some changes tomorrow-ish.